DNN Platform/Library/Common/Globals.cs

```diff
@@ -369,13 +364,8 @@ public static string ApplicationPath
         /// <value>
         /// The application map path.
         /// </value>
-        public static string ApplicationMapPath
-        {
-            get
-            {
-                return _applicationMapPath ?? (_applicationMapPath = GetCurrentDomainDirectory());
-            }
-        }
+        [Obsolete("Deprecated in 9.7.1. Scheduled for removal in v11.0.0, use DotNetNuke.Abstractions.IApplicationStatusInfo instead.")]

```

Comment:
@bdukes @mitchelsellers 

I'm thinking of changing the verbiage of this to give better instructions. Do you have any thoughts on my suggested change?
```suggestion
        [Obsolete("Deprecated in 9.7.1. Using Dependency Injections resolve 'DotNetNuke.Abstractions.IApplicationStatusInfo' instead. Scheduled for removal in v11.0.0.")]
```

If this format is good, I'll update all the deprecation notices in this PR

Reply:
This has been fixed in the latest code push, resolving conversation